KPI/I.adoc

```diff
@@ -1,45 +1,85 @@
+== I. Graphic overview for non bulletins metadata records
 
-== I. GraphicOverview for non bulletins metadata records
+=== WMCP element(s)
 
-A thumbnail preview image for each available products improves greatly the user experience, allowing the user to recognise or understand by just looking at the thumbnail image, the product she is looking at in the search results. It makes also the WIS portals more user friendly and attractive to the users.
+* `//gmd:identificationInfo/gmd:MD_DataIdentification/gmd:graphicOverview/gmd:MD_BrowseGraphic/gmd:fileName/gco:CharacterString`
+* `//gmd:identificationInfo/gmd:MD_DataIdentification/gmd:graphicOverview/gmd:MD_BrowseGraphic/gmd:filename/gmx:Anchor/@xlink:href`
 
-Examples of catalogues using such information are here:
-
-- https://gisc.dwd.de[GISC DWD]
-
-- https://navigator.eumetsat.int/search?query=MSG%20RGB[EUMETSAT Product Navigator]
-
-
-=== I.1 To which WMCP element(s) does it apply ?
-
-This KPI applies to //<gmd:identificationInfo>/<gmd:MD_DataIdentification>/<gmd:graphicOverview>/<gmd:MD_BrowseGraphic/<gmd:fileName>
-
-....
+```xml
 <gmd:graphicOverview>
   <gmd:MD_BrowseGraphic>
      <gmd:fileName>
         <gco:CharacterString>https://navigator.eumetsat.int/preview/meteosat-msg_naturalenhncd.jpg</gco:CharacterString>
```

Comment:
We had discussions during previous meetings not to put URL in character strings .  I though we agreed to recommend the usage of Anchor's only for URL's.

Reply:
fixed.